storage-access.bs

```diff
@@ -4,27 +4,111 @@ Shortname: storage-access
 Repository: privacycg/storage-access
 URL: https://privacycg.github.io/storage-access/
 Editor: John Wilander, w3cid 89478, Apple Inc. https://apple.com/, wilander@apple.com
-Abstract: TODO
-Status Text: This specification is intended to be merged into the HTML Living Standard. It is neither a WHATWG Living Standard nor a W3C standard.
+Abstract: The Storage Access API enables content in iframes to request access to cookies.
```

Comment:
WebKit's implementation only covers cookies but I believe Mozilla's covers other means of storage. Maybe go with "website data such as cookies"?

Reply:
Fixed in 4ccd693.